src/parser/tests.rs

```diff
@@ -504,6 +505,42 @@ pub fn testing_block() {
     };
 }
 
+#[test]
+pub fn test_global_defs() {
```

Comment:
You can also add some asserts here that makes sure you actually read in the correct number of globals.